src/Generator.Tests/AST/TestAST.cs

```diff
@@ -281,5 +281,12 @@ public void TestMacroLineNumber()
         {
             Assert.AreEqual(98, AstContext.FindClass("HasAmbiguousFunctions").First().Specifiers.Last().LineNumberStart);
         }
+
+        [Test]
+        public void TestImplicitDeclaration()
+        {
+            Assert.AreEqual(false, AstContext.FindFunction("testExplicitFunc").Single().IsImplicit);
+            Assert.AreEqual(true, AstContext.FindFunction("testImplicitFunc").Single().IsImplicit);
```

Comment:
This test is failing. Am I missing something about Implicit and Explicit declarations?